README.md

```diff
@@ -29,7 +30,16 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name, assignment) {
+  this.name = name;
+  this.assignment = assignment;
+  this.givesHomework = function() {
+    console.log(this.name + " gives student " +this.assignment+ " for Friday's homework.")
+  }
+}
+
+var Robin = new Instructor("Robin", "Intro to Ruby");
+Robin.givesHomework();
```

Comment:
Almost! [Check the solution.](https://github.com/ga-wdi-quizzes/week-04/tree/solution#question-2) This would print "Robin gives student null for Friday's homework"